DigitalOffice/Tests/TimeManagementServiceTests/Validators/CreateLeaveTimeRequestValidatorTests.cs

```diff
@@ -0,0 +1,194 @@
+using FluentValidation;
+using FluentValidation.TestHelper;
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Validators;
+using LT.DigitalOffice.TimeManagementService.Enums;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Validators
+{
+    public class CreateLeaveTimeRequestValidatorTests
+    {
+        private Mock<ILeaveTimeRepository> repositoryMock;
+
+        private IValidator<CreateLeaveTimeRequest> validator;
+
+        private CreateLeaveTimeRequest request;
+        private DbLeaveTime expectedDbLeaveTime;
+
+        [SetUp]
+        public void Setup()
+        {
+            repositoryMock = new Mock<ILeaveTimeRepository>();
+
+            validator = new CreateLeaveTimeRequestValidator(repositoryMock.Object);
+
+            request = new CreateLeaveTimeRequest
+            {
+                LeaveType = LeaveType.SickLeave,
+                Comment = "I have a sore throat",
+                StartTime = new DateTime(2020, 7, 24),
+                EndTime = new DateTime(2020, 7, 27),
+                WorkerUserId = Guid.NewGuid()
+            };
+
+            expectedDbLeaveTime = new DbLeaveTime
+            {
+                WorkerUserId = request.WorkerUserId,
+                StartTime = request.StartTime,
+                EndTime = request.EndTime,
+                Comment = request.Comment,
+                LeaveType = request.LeaveType
+            };
+
+            repositoryMock.Setup(x => x.GetUserLeaveTimes(request.WorkerUserId))
+                .Returns(new List<DbLeaveTime> { expectedDbLeaveTime });
+        }
+
+        [Test]
+        public void SuccessfulWorkTimeValidation()
+        {
+            repositoryMock.Setup(x => x.GetUserLeaveTimes(It.IsAny<Guid>()))
+                .Returns(new List<DbLeaveTime>());
+
+            validator.TestValidate(request).ShouldNotHaveAnyValidationErrors();
+        }
+
+        #region WorkerUserId
+        [Test]
+        public void FailValidationEmptyWorkerUserId()
+        {
+            var workerUserId = Guid.Empty;
+            repositoryMock.Setup(x => x.GetUserLeaveTimes(It.IsAny<Guid>()))
```

Comment:
Add empty line

Reply:
what for?

Reply:
It seems to me both of these things - arrange